### PR TITLE
create ce activity.log

### PR DESCRIPTION
**Pull request by sns-seb**
* SONAR-7966 Deprecate "logs" attribute in /api/ce/[task/component/activity]
* SONAR-7834 Remove WebService api/ce/logs
* SONAR-7833 Compute Engine logs task's start/stop and intermediary timings in ce_activity.log only
* SONAR-7841 Compute Engine now logs into sonar.log rather than Task specific log file
* SONAR-7960 Add task UUID to every log of the Compute Engine

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/ce/ws/ComponentAction.java [hunk 1]
```diff
@@ -53,7 +53,8 @@ public ComponentAction(UserSession userSession, DbClient dbClient, TaskFormatter
   public void define(WebService.NewController controller) {
     WebService.NewAction action = controller.createAction("component")
       .setDescription("Get the pending tasks, in-progress tasks and the last executed task of a given component " +
-        "(usually a project). Requires the administration permission on the component.")
+        "(usually a project). Requires the administration permission on the component.<br/>" +
+        "Since 6.1, field \"logs\" is deprecated and its value will always be false")
```
Comment (julienlancelot) [inline server/sonar-server/src/main/java/org/sonar/server/ce/ws/ComponentAction.java]: You need to remove "logs" fields from response example

File: server/sonar-server/src/main/java/org/sonar/server/ce/ws/TaskAction.java [hunk 2]
```diff
@@ -57,7 +57,8 @@ public TaskAction(DbClient dbClient, TaskFormatter wsTaskFormatter, UserSession
   public void define(WebService.NewController controller) {
     WebService.NewAction action = controller.createAction(ACTION)
       .setDescription("Give Compute Engine task details such as type, status, duration and associated component.<br />" +
-        "Requires 'Administer System' or 'Execute Analysis' permission.")
+        "Requires 'Administer System' or 'Execute Analysis' permission.<br/>" +
+        "Since 6.1, field \"logs\" is deprecated and its value will always be false")
```
Comment (julienlancelot) [inline server/sonar-server/src/main/java/org/sonar/server/ce/ws/TaskAction.java]: You need to remove "logs" fields from response example

File: server/sonar-server/src/main/java/org/sonar/server/ce/ws/ComponentAction.java (hunk 1 above)
Comment (julienlancelot) [inline server/sonar-server/src/main/java/org/sonar/server/ce/ws/ComponentAction.java]: You need to remove "logs" fields from response example

File: server/sonar-server/src/main/resources/org/sonar/server/ce/ws/component-example.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
       "componentQualifier": "TRK",
       "status": "PENDING",
       "submittedAt": "2015-09-21T19:28:54+0200",
-      "logs": true
+      "logs": false
```
Comment (julienlancelot) [inline server/sonar-server/src/main/resources/org/sonar/server/ce/ws/component-example.json]: Remove this

File: server/sonar-server/src/main/resources/org/sonar/server/ce/ws/component-example.json [hunk 4]
```diff
@@ -25,6 +25,6 @@
     "startedAt": "2015-09-21T19:25:57+0200",
     "finishedAt": "2015-09-21T19:25:58+0200",
     "executionTimeMs": 1371,
-    "logs": true
+    "logs": false
```
Comment (julienlancelot) [inline server/sonar-server/src/main/resources/org/sonar/server/ce/ws/component-example.json]: Remove this

File: server/sonar-server/src/main/resources/org/sonar/server/ce/ws/task-example.json [hunk 5]
```diff
@@ -12,6 +12,6 @@
     "startedAt": "2015-10-02T11:32:16+0200",
     "executedAt": "2015-10-02T11:32:22+0200",
     "executionTimeMs": 5286,
-    "logs": true
+    "logs": false
```
Comment (julienlancelot) [inline server/sonar-server/src/main/resources/org/sonar/server/ce/ws/task-example.json]: Remove this

**Comment by julienlancelot** [comment]
Minor feedbacks to apply, then LGTM !

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [TaskFormatter.java#L51](https://github.com/SonarSource/sonarqube/blob/13ec75a86d6ccafa8b087ccfba9bbaabefb13490/server/sonar-server/src/main/java/org/sonar/server/ce/ws/TaskFormatter.java#L51): Remove this unused "ceLogging" private field. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1068)
1. MAJOR [TaskFormatterTest.java#L46](https://github.com/SonarSource/sonarqube/blob/13ec75a86d6ccafa8b087ccfba9bbaabefb13490/server/sonar-server/src/test/java/org/sonar/server/ce/ws/TaskFormatterTest.java#L46): Remove this unused "TemporaryFolder". [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2924)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [TaskFormatter.java#L51](https://github.com/SonarSource/sonarqube/blob/7730cf3831bc22de8a01a45c68cfac9548bad906/server/sonar-server/src/main/java/org/sonar/server/ce/ws/TaskFormatter.java#L51): Remove this unused "ceLogging" private field. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1068)
1. MAJOR [TaskFormatterTest.java#L46](https://github.com/SonarSource/sonarqube/blob/7730cf3831bc22de8a01a45c68cfac9548bad906/server/sonar-server/src/test/java/org/sonar/server/ce/ws/TaskFormatterTest.java#L46): Remove this unused "TemporaryFolder". [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2924)

**Pull request merged**